content/en/real_user_monitoring/android/mobile_vitals.md

```diff
@@ -15,13 +15,17 @@ further_reading:
 
 Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android Vitals][1], that can help compute insights about your mobile application's responsiveness, stability, and resource consumption. Mobile Vitals range from poor, moderate, to good.
 
-Mobile Vitals appear on your your application's **Performance Overview** page when you navigate to **UX Monitoring > Performance Monitoring** and click your application. From the mobile performance dashboard for your application, click on a graph in **Mobile Vitals** to apply a filter by version or examine filtered sessions.
+You can view Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Monitoring** and clicking your application.
```

Comment:
Could we use "selecting" instead of "clicking" when we refer to choosing an application? I think that might be clearer since the applications are in a drop down menu

Reply:
Yes! Will make that change.